tcms/settings/common.py

```diff
@@ -461,3 +461,6 @@
 }
 
 TEMP_DIR = pathlib.Path(tempfile.gettempdir())
+
+# HACK: See #2123,
```

Comment:
this is not a hack, but at present a required configuration. add the link to the upstream django-grapelli issue and if you can try adding this change into a separate commit b/c it is independent of the tests themselves (but related.)